.github/workflows/release-branch.yml

```diff
@@ -2,18 +2,128 @@ name: Create Release Branch
 # creates a release branch, creates a PR and bumps all versions as specified
 on:
   workflow_dispatch:
-    inputs:
-      changes:
-        description: |
-          A string of charts and their required update, e.g. "icm-as:minor icm-web:patch".
-          Dependent charts don't have to be specified and will be bumped automatically.
-          This action cannot be used for the iom charts, since they don't use bump2version.
-        required: true
-        type: string
-
 
 jobs:
+  find-charts-to-release:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.matrix.outputs.matrix }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4.1.1
+      with:
+        fetch-depth: 0
+    - name: Get last SHA with tag
+      id: last_tagged_sha
+      run: |
+        git config user.name "$GITHUB_ACTOR"
+        git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+        
+        if [[ ${{ github.ref_name }} = develop/* ]]; then
+          IFS='/' read -ra my_array <<< "${{ github.ref_name }}"
+          
+          last_tag=$(git describe --tags --match="${my_array[1]}-*" --abbrev=0 HEAD)
+          sha=$(git rev-list -n 1 $last_tag)
+          
+          base_sha=$(git rev-parse HEAD)
+        
+          echo "sha=$sha" >> $GITHUB_OUTPUT
+          echo "base_sha=$base_sha" >> $GITHUB_OUTPUT
+        else
+          echo "Only development branches could be released" 1>&2
+          exit 1
+        fi
+
+    - name: Get list of changed charts
+      id: list-changed-charts
+      uses: tj-actions/changed-files@v40.1.1
+      with:
+        dir_names_max_depth: 2
```

Comment:
Why is this limitation introduced?

Reply:
Yes you are right. This only need is directories are delivered.